bin/update-yarn

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# inspired by https://gist.github.com/maxrimue/16e272c4e0b8a8ac9dd5570e180e08c0
+
+# this script is only for CircleCI
+$CIRCLECI || exit 0
+
+if [[ $CIRCLE_BRANCH != *"greenkeeper"* ]]; then
+	# Not a GreenKeeper Pull Request, aborting
+	exit 0
```

Comment:
This tests that we are looking at a greenkeeper like branch so we don't do this on anything else.